app/managers.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) "
+            f"VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*actor) for actor in actors_cursor]
+
+    def update(self, actor_id: int, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
+            f"SET first_name=?, last_name=? "
+            f"WHERE id=?",
+            (first_name, last_name, actor_id)
+        )
+        self._connection.commit()
+
+    def delete(self, actor_id: int) -> None:
+        self._connection.execute(
+            f"DELETE FROM {self.table_name} "
+            "WHERE id=?",
+            (actor_id,))
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well